Dockerfile

```diff
@@ -158,14 +162,12 @@ RUN --mount=type=cache,target=/root/.cache/microdnf:rw \
 # need to upgrade pip and install wheel before installing grpcio, before installing tensorflow on aarch64
 # use caching to speed up multi-platform builds
 ENV PIP_CACHE_DIR=/root/.cache/pip
-RUN --mount=type=cache,target=/root/.cache/pip \
-    pip install --upgrade pip && \
-    pip install wheel && \
```

Comment:
Is it not needed anymore?

Reply:
it is needed only for arm builds, as we do not support it yet, we can remove it. (but on my laptop, the build has worked without it, arm here.)

Reply:
I guess multi-arch build may become broken if this is needed for arm.... upstream is doing those multi-arch builds.

Reply:
there is no intention to push these changes upstream. Do you think it is still necessary to re-add it?

Reply:
And, as I mentioned, removing it didn't break the build on my laptop, which is arm-based.

Reply:
IMO, the changes in this PR should be done in a way that are compatible with upstream, so that we can brought these changes there and reduce diversion.

Konflux just needs a `requirements.txt` file. I think we can prepare a PR that can fulfill this need and that we can bring to upstream.

That said... If removing these is not breaking the build, I'm OK with removing. But we should verify that upstream build won't break.

Reply:
great, please take a look in the latest changes.